themes/default/templates/home.html

```diff
@@ -237,10 +246,28 @@ <h2 class="panel-heading">{{title}}</h2>
 
     $('#to-french').click(function () {
       document.cookie = 'language=fr';
+      location.reload();
     });
     $('#to-english').click(function () {
       document.cookie = 'language=en';
+      location.reload();
```

Comment:
@willmero wondering what the location reload is for?

Reply:
@mwraich This is to trigger a reload of the page after changing the cookie. It will re-request the page, thus triggering the other language to be reloaded (this gets around the complexities of engineering a hot-reload function, one line of code is better than one-hundred :D )

Reply:
@willmero looks good to me!